ledger/eval.go

```diff
@@ -1256,7 +1260,7 @@ func eval(ctx context.Context, l ledgerForEvaluator, blk bookkeeping.Block, vali
 		txvalidator.ctx = validationCtx
 		txvalidator.txgroups = paysetgroups
 		txvalidator.done = make(chan error, 1)
-		go txvalidator.run()
+		go txvalidator.run(l)
```

Comment:
kind of weird that it's the only argument passed as a function parameter.. but it's legit.

Reply:
good catch, moved to the `txvalidator` construction code just above